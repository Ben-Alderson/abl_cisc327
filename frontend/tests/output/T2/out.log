Enter command: ERROR: Invalid command
Enter command: ERROR: Invalid command
Enter command: ERROR: You must login before you logout
Enter command: 