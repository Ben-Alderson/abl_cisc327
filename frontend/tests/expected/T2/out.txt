Enter command: Enter login: ERROR: Invalid login
Enter command: ERROR: You must login before you logout
Enter command: 